numpy/linalg/_linalg.py

```diff
@@ -3679,3 +3684,264 @@ def vecdot(x1, x2, /, *, axis=-1):
 
     """
     return _core_vecdot(x1, x2, axis=axis)
+
+# Generalized eigenvalue problem
+def _geneigvals_dispatcher(A, B):
+    return (A, B)
+
+@array_function_dispatch(_geneigvals_dispatcher)
+def geneigvals(A, B):
```

Comment:
How do you feel about making the parameters positional-only?

```suggestion
def _geneigvals_dispatcher(A, B, /):
    return A, B

@array_function_dispatch(_geneigvals_dispatcher)
def geneigvals(A, B, /):
```